packages/server/src/auth/utils/cookies.ts

```diff
@@ -39,3 +67,12 @@ function parseCookieHeader(cookieHeader: string): Record<string, string> {
         {} as Record<string, string>
     );
 }
+
+function createCookieString(cookieOptions: CookieOptions): string {
+    const expiresInUtc = cookieOptions.expiresAt ? new Date(cookieOptions.expiresAt).toUTCString() : "";
+    let cookieString = `${cookieOptions.name}=${cookieOptions.value}; path=/; SameSite=Lax;`;
+    if (expiresInUtc) {
+        cookieString += ` expires=${expiresInUtc};`;
+    }
+    return cookieString;
```

Comment:
I know there's URLSearchParams that does this work for query params, there's nothing similar to use, right?

Reply:
Sadly, not that I know of. The idea is to follow this pattern with other properties like `Secure` and `HttpOnly` once we also support more secure cookies

Reply:
I could create my own util though in the httponly pr as we'll have more options to add there